sensors/test_deps.go

```diff
@@ -21,30 +21,28 @@ const (
 	BadTime = "NOT A TIME"
 )
 
-// SetupDeps returns the dependencies based on the sensors passed as arguments.
-func SetupDeps(sensors []string) resource.Dependencies {
+// SetupDeps returns the dependencies based on the lidar passed as argument.
+func SetupDeps(lidarName string) resource.Dependencies {
 	deps := make(resource.Dependencies)
-
-	for _, sensor := range sensors {
-		switch sensor {
-		case "good_lidar":
-			deps[camera.Named(sensor)] = getGoodLidar()
-		case "warming_up_lidar":
-			deps[camera.Named(sensor)] = getWarmingUpLidar()
-		case "replay_sensor":
-			deps[camera.Named(sensor)] = getReplaySensor(TestTime)
-		case "invalid_replay_sensor":
-			deps[camera.Named(sensor)] = getReplaySensor(BadTime)
-		case "invalid_sensor":
-			deps[camera.Named(sensor)] = getInvalidSensor()
-		case "gibberish":
-			return deps
-		case "finished_replay_sensor":
-			deps[camera.Named(sensor)] = getFinishedReplaySensor()
-		default:
-			continue
```

Comment:
why was this default removed?

Reply:
if this change wasn't needed to implement the PR, please add it back

Reply:
It used to be in a for statement, which is why we could have a continue (the for loop was needed because it used to be a list of sensors). Since we no longer have a list, we cannot call continue. Is there something you'd prefer me to put as default?